app/views/NextUpdates.scala.html

```diff
@@ -44,32 +49,29 @@ <h1 id="page-heading" class="heading-xlarge"> @messages("nextUpdates.heading")</
 		</summary>
 
 		<div class="govuk-details__text" id="details-content-0" data-module="govuk-details" aria-hidden="false">
-			<h2 id="quarterlyReturns-heading" class="heading-medium">@messages("obligations.quarterlyUpdates")</h2>
-			<p><span id="quarterly-dropdown-line1">@messages("nextUpdates.dropdown.quarterlyReturn.text")</span> </p>
-			<p><span id="quarterly-dropdown-line2">@Html(messages("nextUpdates.dropdown.quarterlyReturn.text.lin2",
-			))
-			</span>
-			</p>
+			<h2 id="quarterlyReturns-heading" class="govuk-heading-m">@messages("obligations.quarterlyUpdates")</h2>
+			@p(id = Some("quarterly-dropdown-line1"), content = Html(messages("nextUpdates.dropdown.quarterlyReturn.text")))
```

Comment:
Note: To my mind having the component like `p` which requires to write `Html(...)` in the view is evil. I would say we better make the `p.apply(...)` curried and the content html is better to be passed there solely in a last argument list, like:
```
@p(id = Some("quarterly-dropdown-line1")) { 
    @messages("nextUpdates.dropdown.quarterlyReturn.text")
}
```

Reply:
Not needed to be solved right now, just a side note to think about, no worries.